lang/src/main/java/com/sap/psr/vulas/goals/AbstractGoal.java

```diff
@@ -543,10 +543,10 @@ public String toJson() throws IllegalStateException {
 		// Goal configuration
 		b.append(",\"configuration\":[");
 		int c = 0;
-		final Iterator<String> iter = this.getConfiguration().getConfiguration().subset("vulas").getKeys();
+		final Iterator<String> iter = this.getConfiguration().getConfiguration().getKeys("vulas");
 		while(iter.hasNext()) {
 			final String key = iter.next();
-			final String[] value = this.getConfiguration().getConfiguration().getStringArray("vulas." + key);
+			final String[] value = this.getConfiguration().getConfiguration().getStringArray(key);
```

Comment:
This change results in a different name for the keys of all the vulas related configurations as they will now maintain the prefix "vulas.". The newly generated JSON now looks like {"source":"GOAL_CONFIG","name":"vulas.report.exceptionScopeBlacklist","value":"TEST,PROVIDED"} whereas until now we had name:"report.exceptionScopeBlacklist", which is the same value we have in the database. I would keep the old format or do we have some other reason to change it?

Reply:
I always found it inconsistent that the parameter names are not properly stored in the database (vulas.foo.* in the client, and foo.* in the database and UI). Instead of going back to the old way, could we add some backward compatible logic (that can be removed later on together with the now deprecated old format for exemptions)?

Reply:
I always found it inconsistent that the parameter names are not properly stored in the database (vulas.foo.* in the client, and foo.* in the database and UI). Instead of going back to the old way, could we add some backward compatible logic (that can be removed later on together with the now deprecated old format for exemptions)?

Reply:
I agree, we can use the parameters as is and handled backward compatibility with the values in the backend. I will add it.